### PR TITLE
[Console] add documentation for kernel-aware console applications

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
See https://github.com/symfony/symfony/pull/21019

### CODE REVIEW

File: components/console/single_command_tool.rst [hunk 1]
```diff
@@ -54,3 +54,33 @@ Of course, you can still register a command as usual::
...
+  $application = new Application($kernel);
+  $command = new DefaultCommand();
+
+  $application->add($command);
+
+  $application->setDefaultCommand($command->getName(), true);
+  $application->run();
+
+an instance of ``Symfony\Bundle\FrameworkBundle\Console\Application`` will automatically
+load all default commands. If you want to change this behaviour, you can pass ``false``
```
Comment (mickaelandrieu) [inline components/console/single_command_tool.rst]: I think you can put this information in a note instead, wdyt ?

Comment (ShinDarth) [inline components/console/single_command_tool.rst]: @mickaelandrieu done

**Review by Jean85 — APPROVED** [review]

**Review by Jean85 — APPROVED** [review]
Reviewed

**Comment by OskarStark** [comment]
Code PR was closed

**Pull request closed without merging**